src/util/drawLink.js

```diff
@@ -29,11 +30,12 @@ export default function (linkAnchorPoints, refPoint, boundingBox, context, color
   const end = external.cornerstoneMath.point.findClosestPoint(boundingBoxPoints, start);
 
   // Finally we draw the dashed linking line
-  context.beginPath();
-  context.strokeStyle = color;
-  context.lineWidth = lineWidth;
-  context.setLineDash([2, 3]);
-  context.moveTo(start.x, start.y);
-  context.lineTo(end.x, end.y);
-  context.stroke();
+  const lineDash = [2, 3];
+
+  path(context, { color,
```

Comment:
Could we make the options into a separate variable? Seems like it might be cleaner to read

Reply:
Yep, I agree it would make it clearer. It might be easier to do that clean up after the next couple of PRs, which will convert many of these `path` calls into calls to things like `drawLine`. Also, a lot of these option objects will reduce down to `{ color }` once we take account the default value of `lineWidth`.